### ISSUE TITLE
Aiming Problems

### ISSUE THREAD

**Issue by KomariVGC** [Loara228/deadlock-esp#59]
After the 11-21 update, it seems that you can't target creeps and souls correctly

**Comment by erxson** [comment]
делаем бля парень 
https://github.com/Loara228/deadlock-esp/pull/60

**Comment by KomariVGC** [comment]
I don't know why, aiming has no effect on new heroes, but esp does.

**Comment by Loara228** [comment]
ESP can draw an unknown hero. Aim cannot work correctly with an unknown hero.
https://github.com/Loara228/deadlock-esp/blob/3700bf7144d43e07277b8c8bbbbf76f3d925ba49/src/external/interfaces/structs.rs#L265C1-L265C40

New heroes are not processed in code.
https://github.com/Loara228/deadlock-esp/blob/3700bf7144d43e07277b8c8bbbbf76f3d925ba49/src/external/interfaces/enums.rs#L120

**Comment by malwaredotid** [comment]
so basically we just add new hero in the enums.rs and add new icons for the new heroes Aim will gonna work again ?

**Comment by Loara228** [comment]
yep, but also u need to add bones [here](https://github.com/Loara228/deadlock-esp/blob/3700bf7144d43e07277b8c8bbbbf76f3d925ba49/src/external/interfaces/enums.rs#L21).
The compiler can help you make a couple more edits.

**Comment by KomariVGC** [comment]
I added new hero information, maybe the value is wrong? esp icon works but
aiming doesn't work

malwaredotid ***@***.***> 于 2024年11月23日周六 18:40写道：

**Comment by malwaredotid** [comment]
yeah i think so, i dont know to find the value of the heroes

**Comment by Loara228** [comment]
@stanleyhuangQAQ, try to sync with the newest version
[commit]https://github.com/Loara228/deadlock-esp/commit/3700bf7144d43e07277b8c8bbbbf76f3d925ba49
